test/simplifyExpression/basicsSearch/mixedNumber.test.js

```diff
@@ -0,0 +1,103 @@
+const convertMixedNumberToImproperFraction = require(
+  '../../../lib/simplifyExpression/basicsSearch/convertMixedNumberToImproperFraction');
+const mixedNumber = require('../../../lib/simplifyExpression/basicsSearch/mixedNumber');
+const TestUtil = require('../../TestUtil');
+
+function testIsMixedNumber(input, output) {
+  TestUtil.testBooleanFunction(mixedNumber.isMixedNumber, input, output);
+}
+
+function testIsNegativeMixedNumber(input, output) {
+  TestUtil.testBooleanFunction(mixedNumber.isNegativeMixedNumber, input, output);
+}
+
+function testGetWholeNumber(input, output) {
+  input = TestUtil.flattenAndParse(input);
+  TestUtil.testFunctionOutput(mixedNumber.getWholeNumber, input, output);
+}
+
+function testGetNumerator(input, output) {
+  input = TestUtil.flattenAndParse(input);
+  TestUtil.testFunctionOutput(mixedNumber.getNumerator, input, output);
+}
+
+function testGetDenominator(input, output) {
+  input = TestUtil.flattenAndParse(input);
+  TestUtil.testFunctionOutput(mixedNumber.getDenominator, input, output);
+}
+
+function testConvertMixedNumberToImproperFraction(exprString, outputList, outputStr) {
+  TestUtil.testSubsteps(convertMixedNumberToImproperFraction, exprString, outputList, outputStr);
+}
+
+describe('isMixedNumber', function () {
+  const tests = [
+    ['5(1)/(6)', true],
+    ['19(2)/(3)', true],
+    ['-1(7)/(8)', true],
+    ['4*(1/2)', false],
+    ['(1/2)3', false],
+    ['3*10/15', false],
```

Comment:
actually here's a followup though @aliang8 @ldworkin 

This change is very specific for Socratic's use case. Does it make sense for when people actually enter that text? If you were just going to mathsteps and entering `-1(7)/(8)` ... I guess most people wouldn't do that :p 

But I think that very Socratic focused changes could happen in Athena. e.g. you could convert `-1(7)/(8)` to `-1 + 7/8` and just pass that into mathsteps. Just a thought - lemme know what you think about this kind of thing in general for dealing with future parsing errors